lib/tpm_kdfa.c

```diff
@@ -51,6 +51,34 @@ static const EVP_MD *tpm_algorithm_to_openssl_digest(TPMI_ALG_HASH algorithm) {
     /* no return, not possible */
 }
 
+static HMAC_CTX *hmac_alloc()
+{
```

Comment:
@williamcroberts sigh, I thought I checked it followed the coding standards but I may had missed when re-spinning the commit, sorry about that. I'll be more careful next time.

Thanks for picking them!